Core/source/assets/flowcrypt-ios-begin.js

```diff
@@ -35,8 +35,8 @@ const window = { navigator, crypto, setTimeout, clearTimeout };
 const self = { window, navigator, crypto };
 global.window = window;
 
-var engine_host_cb_value_formatter = function(val) {
-    coreHost.handleCallback(val.json, val.data);
+var engine_host_cb_value_formatter = function(key, val) {
+    coreHost.handleCallback(key, val.json, val.data);
 };
```

Comment:
please call it `callbackId` instead of `key`.

Since we deal with encryption keys everywhere, I'm trying to avoid the word in non-cryptographic contexts.